guides/v2.2/javascript-dev-guide/widgets/widget-row-builder.md

```diff
@@ -0,0 +1,267 @@
+---
+group: javascript-developer-guide
+subgroup: 3_Widgets
+title: RowBuilder widget
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+## Overview
```

Comment:
This heading is unnecessary. The first paragraph of a topic is generally accepted to be the overview.